AI/scripts/deploy_to_server.py

```
@@ -1,173 +1,279 @@
-        "name"      : "TCN",
-        "local_dir" : os.path.join(project_root, "AI/data/weights/tcn"),
-        "remote_dir": f"{SERVER_WEIGHTS_PATH}/tcn",
-        "files"     : ["model.pt", "scaler.pkl", "metadata.json"],
-    },
-]
-
-
-def create_ssh_client() -> paramiko.SSHClient:
-    """SSH 연결 생성 (키 문자열로 직접 연결)"""
     ssh = paramiko.SSHClient()
     ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
```

Ruff (0.15.12)

[error] Paramiko call with policy set to automatically trust the unknown host key

(S507)